google/cloud/forseti/services/inventory/storage.py

```diff
@@ -660,7 +660,7 @@ def _get_parent_name(asset_pb):
             return '/'.join(asset_pb.name.split('/')[:-4])
 
         elif (asset_pb.asset_type.startswith('google.appengine') or
-              asset_pb.asset_type.startswith('google.bigquery') or
+              asset_pb.asset_type.startswith('google.cloud.bigquery') or
```

Comment:
Minor nit: Can we make these alphabetical?

Reply:
Done.